docs/OPERATOR_GETTING_STARTED.md

```diff
@@ -76,74 +83,150 @@ $ ./install.sh
 
 ### 4. Generate Operator Keys
 
-The following command will generate your operator's public and private keys (appear as "pk" and "sk" in the output). 
+#### Password file
 
-```
-$ docker run --rm -it 'bloxstaking/ssv-node:latest' /go/bin/ssvnode generate-operator-keys
-```
+You will need to create a file (named `password` in this example) containing the password you chose for your Secret Key:
 
-### 5. Create a Configuration File
+```bash
+echo "<MY_OPERATOR_PASSWORD>" >> password
+```
 
-Fill all the placeholders (e.g. `<ETH 2.0 node>` or `<db folder>`) with actual values,
-and run the command below to create a `config.yaml` file.
+#### Key pair generation and encryption
 
+The node Docker image will generate keys for you, then encrypt them with a password you provide, using the following command:
 
+```bash
+docker run --name ssv-node-key-generation -v "$(pwd)/password":/password -it "bloxstaking/ssv-node:latest" /go/bin/ssvnode generate-operator-keys --password-file=password && docker cp ssv-node-key-generation:/encrypted_private_key.json ./encrypted_private_key.json && docker rm ssv-node-key-generation
```

Comment:
I saw some of this repeated in the `DEV_GUIDE` file, maybe we could take it to separate file and link it in both?